models/app.js

```diff
@@ -49,27 +48,20 @@ function createModel (cb) {
     const dir = files[0]
     manager.create(dir, done)
   })
+
   model.effect('delete', (state, data, send, done) => {
     const dat = data
     manager.remove(dat, done)
   })
+
+  // copy a dat share link to clipboard and open a modal
```

Comment:
there's nothing wrong with comments per se, it's just that if code can be written in a way that comments aren't necessary, that's even better